libs/SalesforceSDKCore/SalesforceSDKCore/Classes/Security/SFUserAccountManager.m

```diff
@@ -470,6 +366,26 @@ - (NSString*)uniqueUserAccountIdentifier {
     return identifier;
 }
 
+-(SFOAuthCredentials *)currentCredentials {
```

Comment:
I'm not immediately seeing why we're separately managing a set of "current credentials", as opposed to simply managing the credentials that are part of the `currentUser` account—this is a practice we're trying to get away from.  Same for `currentUserIdentity`: why is that information not accessed maintained as encapsulated data within the current user account itself?